pinot-broker/src/main/java/org/apache/pinot/broker/broker/helix/BaseBrokerStarter.java

```diff
@@ -409,7 +409,8 @@ public void start()
         _brokerConf.getProperty(CommonConstants.Broker.CONFIG_OF_ENABLE_THREAD_ALLOCATED_BYTES_MEASUREMENT,
             CommonConstants.Broker.DEFAULT_THREAD_ALLOCATED_BYTES_MEASUREMENT));
     Tracing.ThreadAccountantOps.initializeThreadAccountant(
-        _brokerConf.subset(CommonConstants.PINOT_QUERY_SCHEDULER_PREFIX), _instanceId);
+        _brokerConf.subset(CommonConstants.PINOT_QUERY_SCHEDULER_PREFIX), _instanceId,
+        org.apache.pinot.spi.config.instance.InstanceType.BROKER);
```

Comment:
(minor) Replace it with a import

Reply:
This was expanded because of conflict with `org.apache.helix.InstanceType;`